src/main/scala/io/ino/solrs/Utils.scala

```diff
@@ -15,4 +21,13 @@ object Utils {
     }
   }
 
+  def solrUrl(baseUrl: String): String = {
+    val url = new URL(baseUrl)
+    url.getProtocol + "://" +  url.getHost + ":" + url.getPort + "/solr"
+  }
+
+  private val adminRequestTypes = List(classOf[CollectionAdminRequest[CollectionAdminResponse]])
+
+  def isAdminType(request: SolrRequest[_]): Boolean =
+    adminRequestTypes.exists(el => el.isAssignableFrom(request.getClass))
```

Comment:
Because `adminRequestTypes` contains a single element only this can be simplified here.